src/index.ts

```diff
@@ -0,0 +1,61 @@
+import { Cluster } from "puppeteer-cluster";
+
+import { Options, Content } from "./types";
```

Comment:
The `Content` is not used in the file, you can 🔥 it

```suggestion
import { Options } from "./types";
```

Reply:
I added a linter for this stuff 🙏